app/assets/json/translations/react/es.json

```diff
@@ -790,39 +937,68 @@
   "listings.habitat.sweatEquity.p1": "No debe hacer ningún depósito inicial. Sí debe realizar 500 horas de voluntariado. Las horas de voluntariado son horas que usted, sus amigos y su familia trabajan como voluntarios.",
   "listings.habitat.sweatEquity.p2": "Si le preocupa su capacidad de realizar 500 horas de voluntariado, póngase en contacto con Habitat for Humanity Greater San Francisco para solicitar asistencia.",
   "listings.habitat.sweatEquity.title": "Horas de voluntariado",
+  "listings.Hearing impairments.title": "Discapacidad auditiva",
+  "listings.Hearing/Vision (Communication).title": "Audición/visión (comunicación)",
+  "listings.Hearing/Vision impairments.title": "Discapacidad visual o auditiva",
   "listings.householdMaximumIncome": "Ingresos anuales del grupo familiar",
   "listings.housingProgram": "Programa de vivienda",
   "listings.importantProgramRules": "Reglas Importantes del Programa",
+  "listings.includesPriorityUnits": "Incluye unidades con prioridad para %{priorities}",
```

Comment:
content mismatch with the English translation file (applies to other non-english translation files)